src/body/body.rs

```diff
@@ -183,54 +183,44 @@ impl Body {
     }
 
     fn take_delayed_eof(&mut self) -> Option<DelayEof> {
-        self
-            .extra
+        self.extra
             .as_mut()
             .and_then(|extra| extra.delayed_eof.take())
     }
 
     fn extra_mut(&mut self) -> &mut Extra {
-        self
-            .extra
-            .get_or_insert_with(|| Box::new(Extra {
+        self.extra.get_or_insert_with(|| {
+            Box::new(Extra {
                 delayed_eof: None,
                 on_upgrade: OnUpgrade::none(),
-            }))
+            })
+        })
     }
 
     fn poll_eof(&mut self) -> Poll<Option<Chunk>, ::Error> {
         match self.take_delayed_eof() {
-            Some(DelayEof::NotEof(mut delay)) => {
```

Comment:
Just checking, this is simply rustfmt?

Reply:
Ah damn totally could have been forgot to disable it for this PR oops. Would you like me to revert this?